README.md

```diff
@@ -45,18 +49,35 @@ git clone https://github.com/Arcadia-Science/B-SOID.git
 cd /path/to/B-SOID/
 ```
 
-For MacOS users
-```
+For MacOS users (note: you need an x86 archicture MacOS):
+```bash
 conda env create -n bsoid_v2 -f requirements.yaml (macOS)
 ```
 
-or for Windows users
-
+or for Windows users:
+```bash
+conda env create -n bsoid_v2 -f requirements_win.yaml (windows)
 ```
-conda env create -n bsoid_v2 -f requirements_win.yaml (windows) 
+
+or for Linux users (this was tested with Ubuntu version 22.2):
+
+Linux `ffmpeg` distribution on Conda doesn't work (it has missing libraries). Because of this, you need to install `ffmmpeg` separately from the Conda environment setup. First, let's do that:
```

Comment:
```suggestion
Linux `ffmpeg` distribution on Conda doesn't work (it has missing libraries). Because of this, you need to install `ffmpeg` separately from the Conda environment setup. First, let's do that:
```